unityfiles/Assets/Scripts/Helper/Util/WUArrays.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Linq;
+
+using UnityEngine;
+
+
+static class WUArrays
+{
```

Comment:
Declare if public or private, and change name of class to something more readable